modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/js/FileEntryPicker.js

```diff
@@ -36,7 +37,16 @@ const FileNamePicker = ({
 	}, [inputValue]);
 
 	const onInputChange = ({target}) => {
-		if (target.files[0].size > maxFileSize) {
+		const fileType = target.files[0]?.type;
+		const maxMimeSize = Number(maxMimeTypeSize[fileType]);
```

Comment:
`maxMimeSize` sound weird to me, for me `maxFileTypeSize` works better. What do you think? 

If you have another proposal is welcome.

Reply:
@boton 
maxMimeTypeSize is optional configuration. It depends on whether the max size is configurated based on its type or not, so we don't need to add it to proper prop validation.
I think maxFileTypeSize is nice, I updated it to my code